create_cluster.py

```diff
@@ -21,8 +21,38 @@
 from clusters import *
 
 def CreateClusterV1(filename: str,
-                    features: List[str],
-                    duplicate_column_name: str,
+                    features: List[str] = [
+        "AgeAtTestDate",
```

Comment:
Oh, ok.  But this list should be defined outside the function definition so that others can refer to it.  And then function default should be the previously defined name.  OK?  Same with duplicate_column_name.

Reply:
Like a global variable (for both duplicate_column_name and features)? 

There was no default previously. I included default only in this version. Let me know if I should modify this.

Reply:
Yes, a global name for both pieces of information, like you have for the other one... make sure v1 is in the name.  Hopefully similar names.